packages/kbn-journeys/journey/journey_config.ts

```diff
@@ -119,6 +119,11 @@ export interface JourneyConfigOptions<CtxExt> {
    * but before archives are loaded. APM traces are not collected for this hook.
    */
   beforeSteps?: (ctx: BaseStepCtx & CtxExt) => Promise<void>;
+  /**
+   * Use this to define actions that will be executed on teardown,
+   * but before archives are unloaded. APM traces are not collected for this hook.
+   */
+  afterSteps?: (ctx: BaseStepCtx & CtxExt) => Promise<void>;
```

Comment:
Do we still need `afterSteps` since you moved to esArchiver?
I was also thinking that since we re-start servers between journeys, I can't think when `afterSteps` is needed: we want to ingest data to ES one time and use for both WARMUP/TEST phases. Clearing Kibana doesn't makes sense since we restart it anyways.
Do I miss something?

Reply:
You're right, no longer needed! I removed it in e924f98.